src/config/getWebpackClientConfig.js

```diff
@@ -10,7 +10,6 @@ const detectEnvironmentVariables = require("../lib/detectEnvironmentVariables");
 const getWebpackAdditions = require("../lib/getWebpackAdditions").default;
 const {
   additionalLoaders,
-  additionalPreLoaders,
```

Comment:
Yeah now you have to add `enforce: 'pre'` to loader. Preloaders are completely gone https://webpack.js.org/guides/migrating/#module-preloaders-and-module-postloaders-was-removed